lms/djangoapps/instructor_task/api.py

```diff
@@ -280,6 +280,11 @@ def submit_bulk_course_email(request, course_key, email_id):
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
     targets = [target.target_type for target in email_obj.targets.all()]
+    #task_input has a limit to the size it can store, so cohorts are combined and counted
+    num_cohorts = targets.count('cohort')
+    if num_cohorts > 1:
+        targets.remove('cohort')
+        targets.add('{} cohorts'.format(num_cohorts))
```

Comment:
Fair enough. I guess if someone had like 10^200 cohorts this would break, but we'd probably have other problems at that point ;).